plugins/chat/app/services/chat/trash_message.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
```

Comment:
Im tempted to say we should take the channel ID as param too, and fetch the message with this ID, it would limit the risk of given a wrong message ID